src/Microsoft.ML.Transforms/RandomFourierFeaturizing.cs

```diff
@@ -31,9 +31,12 @@
 
 namespace Microsoft.ML.Transforms.Projections
 {
+    /// <summary>
+    /// Maps vector columns to a random low-dimensional feature space.
```

Comment:
>Maps vector columns to a random low-dimensional feature space. [](start = 8, length = 62)

Maps vector columns to a low -dimensional feature space. I'm not expert in RFF, it just word random in from of low-dimensional sounds like we just throw some random numbers in new vector.

Reply:
Same across other places with same description

---
In reply to: [254050520](https://github.com/dotnet/machinelearning/pull/2366#discussion_r254050520) [](ancestors = 254050520)